caktus_primer/templates/styleguide/styleguide-navigation_patterns-search.html

```diff
@@ -0,0 +1,50 @@
+{% extends "styleguide/styleguide.html" %}
```

Comment:
can we simplify the name to "navigation" from "navigation_patterns"

everything in the styleguide is a pattern, really, so its redundant. short and sweet.